### ISSUE TITLE
Mac OS X - Fault with 'Check for Update'...

### ISSUE THREAD

**Issue by Jitsi-Bitsi-Spider** [jitsi/jitsi#217]
Have loved Jitsi from Day One.

So, Hat's off to everybody who's burned through a Keyboard or two, to make all off this possible.

Courtesies aside, here's the lil' Bug - (that I've got?);

- Jitsi is the only Application I've ever used, that fails to determine which version of OS X you are currently using, before offering up a newer version for Installation.

This especially annoying when administering more than 20 Macs running anything from Snow Leopard (v10.6), all the way through to El Capitan (v10.11).

Jitsi will continuously offer up an update, and if accepted by the end-user, will simply blindly download and install this update - immaterial of whether it is compatible with the version of OS X that it is running on.

This issue is further complicated on the Jitsi Download page, by failing to list the actual specific minimum System Requirements - both from a Hardware and OS standpoint.

If the 'old-school' version of Jitsi, (for Snow Leopard (v10.6) and Lion (v10.7) are actually never going to be updated any further, (from a feature-set perspective), then why not at least disable the 'Check for Update' capability within these versions - which allows them to perpetually offer up a newer version of Jitsi, each time one is released…?

In summary;
- Prevent legacy versions of Jitsi from offering up newer, incompatible versions.
- Update the website to include clear and specific, System Requirements.

This aside - Wow.  What a true monster-weapon of an Application…

Warmest Regards - and forever hooked,

Jitsi-Bitsi-Spider
(c;=

**Comment by damencho** [comment]
You can easily workaround this by deleting the following file: /Applications/Jitsi.app/Contents/Resources/Java/lib/native/mac/libsparkle_init.dylib
An option to disable updates will be very handy. 
There is also the option to add some wrong address in your config for the property controlling the place for updates:
net.java.sip.communicator.UPDATE_LINK
But not sure, maybe sparkle will show an error on every start in such situation, just disabling sparkle by deleting its binary will do the trick.

**Comment by ibauersachs** [comment]
The 'legacy' versions of Jitsi shouldn't be used anymore as they contain security issues (as do these old OS X versions). Closing this as the update reminder can be disabled as per damencho's recommendations.
